lib/elixir/lib/process.ex

```diff
@@ -940,6 +940,23 @@ defmodule Process do
   @spec unalias(alias) :: boolean
   defdelegate unalias(alias), to: :erlang
 
+  @doc """
+  Add a descriptive term to the current process.
+  The term does not need to be unique, and in Erlang/OTP 27+ will be shown in Observer.
+
+  ## Examples
+
+      Process.set_label(:worker)
+      #=> :ok
+  """
+  @spec set_label(term()) :: :ok
```

Comment:
```suggestion
  @doc """
  Add a descriptive term to the current process.

  The term does not need to be unique, and in Erlang/OTP 27+ will be shown in Observer.

  ## Examples

      Process.set_label(:worker)
      #=> :ok
  """
  @doc since: "1.17.0"
  @spec set_label(term()) :: :ok
```